neptune/client.py

```diff
@@ -455,6 +455,17 @@ def create_channel(self, experiment, name, channel_type):
         except HTTPConflict:
             raise ChannelAlreadyExists(channel_name=name, experiment_short_id=experiment.id)
 
+    @with_api_exceptions_handler
+    def reset_channel(self, channel_id):
```

Comment:
We should try to let users know wchich channel is "bad", and User never operates on UUIDs

Reply:
`Client` is an internal class. User should not user it directly.